test/bootstrap.js

```diff
@@ -0,0 +1,105 @@
+const test = require('brittle')
+// const { swarm } = require('./helpers')
+const DHT = require('../')
+// const { BOOTSTRAP_NODES } = require('../lib/constants.js')
+
+test('bootstrapper', async function (t) {
+  const node = DHT.bootstrapper(49737)
+
+  await node.ready()
+  t.is(typeof node.address().host, 'string')
+  t.is(typeof node.address().family, 'number')
+  t.is(typeof node.address().port, 'number')
+
+  await node.destroy()
+})
+
+test('local bootstrap with default node settings', async function (t) {
+  const bootstrap1 = new DHT({ bootstrap: [], port: 49737, anyPort: false, ephemeral: false, firewalled: false })
+  await bootstrap1.ready()
+
+  const bootstrap = [bootstrap1.address()]
```

Comment:
don't do this, as address is 0.0.0.0, do { port: bootstrap1.address().port, host: '127.0.0.1' }

Reply:
Yeap, this was already outdated as I learned it today with `dht-rpc`